tasks/shelter/README.md

```diff
@@ -296,334 +296,334 @@ Pay attention to additions:
-- Картинка собаки и отступы могут быть `responsive`. Главное, чтобы картинка была центрирована.
+- One single column. The indents of all elements to the edges of the screen must match. The column itself must be centered. At the same time, both the column itself and the indents to the edges of the screen can be `responsive`.
+- Dog picture and padding can be `responsive`. The main thing is that the picture is centered.
 
 ## width < 320px
 
-Минимальная ширина макета = 320px. После порогового значения расположение верстки значения не имеет. Структура верстки остается такой же, как при ширине 320px. Если в какой-то момент при уменьшении ширины экрана верстка "ломается", это не ошибка.
+Minimum layout width = 320px. After the threshold value, the location of the layout does not matter. The layout structure remains the same as with a width of 320px. If at some point when the screen width is reduced, the layout "breaks", this is not a mistake.
 
 ## Cross-check evaluation criteria. Week 3
 
-❗ **Важное замечание #1**: При изменении ширины с использованием инструментов разработчика (dev tools) и наличии вертикальной полосы прокрутки, расхождение может составлять до 17px (стандартная ширина вертикальной полосы прокрутки). В таком случае надо либо изменить ширину на величину этого расхождения, либо перейти в [task verification features](#task-verification-features), чтобы вертикальной полосы прокрутки не было.  
-❗ **Важное замечание #2**: При масштабировании экрана (например, zoom+ 125%) реальная ширина может отличаться на 1-2 пикселя. Например, реальное значение может быть 767 или 769, хотя инструменты разработчика будут показывать 768. Поэтому надо сдвинуть до точки перехода, несмотря на отличие.  
-❗ **Важное замечание #3**: Единственненный блок, в котором элементы не обязательно должны соблюдать структуру - это блок `Help`. Если элементы идут как перечисляемые элементы, то при проверке на резиновую верстку, отступы от края блока до элементов, а также количество в ряду не учитываются и баллы не снижаются. Также не оценивается, центрируются ли элементы или прибиты к левому краю. Однако нарушение размеров учитывается, будьте внимательны!  
-❗ **Важное замечание #4**: Проверка полного соответствия макетам по Perfect Pixel происходит только на соответствующих контрольных точках - 1280px, 768px, 320px. На всех промежуточных величинах допускается отсутствие точного следования отступам и размерам, но рекомендуется следовать принципам расположения тех или иных элементов, не отмеченных в конкретных требованиях.
+❗ **Important note #1**: When changing the width using the dev tools and having a vertical scrollbar, the discrepancy can be up to 17px (standard vertical scrollbar width). In this case, you must either change the width by the amount of this discrepancy, or go to [task verification features](#task-verification-features) so that there is no vertical scrollbar.
```

Comment:
```suggestion
❗ **Important note #1**: When changing the width using the dev tools and having a vertical scrollbar, the discrepancy can be up to 17px (standard vertical scrollbar width). In this case, you must either change the width by the amount of this discrepancy or go to [task verification features](#task-verification-features) so that there is no vertical scrollbar.

```